docs/conf.py

```diff
@@ -152,13 +152,14 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
-def setup(app):
```

Comment:
I think we want to leave this in?

Reply:
I didn't remove it, I'm just handling it using `html_js_files` instead of `app.add_js_file`.  Both have the same effect, but `html_js_files` accepts multiple paths so it seemed more efficient.

Related documentation:
* [`app.add_js_file`](https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.add_js_file)
* [`html_js_files`](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_js_files)

Reply:
Happy to change it back though if you prefer.

Reply:
Ah, interesting. The new way works for me!